cmd/backup-manager/app/import/import.go

```diff
@@ -97,6 +97,13 @@ func (ro *Options) loadTidbClusterData(restorePath string) error {
 		fmt.Sprintf("--d=%s", restorePath),
 		fmt.Sprintf("--tidb-port=%d", ro.Port),
 	}
+
+	if len(tableFilter) != 0 {
+		for _, tableFilter := range tableFilter {
+			args = append(args, "-f", tableFilter)
+		}
+	}
```

Comment:
```suggestion
	for _, filter := range tableFilter {
		args = append(args, "-f", filter)
	}
```

Reply:
No need to check the length of a slice, right?
And to make the var name more clear.